.readthedocs.yml

```
@@ -2,7 +2,7 @@ build:
   image: latest
 
 python:
-  version: 3.8
+  version: 3.9
```

RTD does not support 3.9
```suggestion
  version: 3.8
```

---

changed